kotlin/build.gradle

```diff
@@ -134,8 +134,15 @@ subprojects {
 
   apply plugin: "org.jlleitschuh.gradle.ktlint"
   apply plugin: "io.gitlab.arturbosch.detekt"
-  afterEvaluate {
+  afterEvaluate { project ->
     project.tasks.findByName('check')?.dependsOn 'detekt'
+
+    project.configurations.configureEach {
+      // No module depends on on Kotlin Reflect directly, but there could be transitive dependencies
+      // in tests with a lower version. This could cause problems with a newer Kotlin version that
+      // we use. To
+      resolutionStrategy.force deps.kotlin.reflect
```

Comment:
To prevent adding the dependency back in the future by accident, can you hard-code the artifact name here and delete it from the map?

Reply:
I thought about this, but personally would have preferred the current way. But I'm changing it.

Reply:
If we actually need the dependency, we can add it back to the list.

Reply:
Did you happen to note which modules include it as a transitive dep? Is dropping it as a direct dependency _actually_ saving us anything?

Reply:
Kotlin Mockito depends on it. Yes, it's saving us a lot, because you need it only for tests. I could have added a dependency on the latest version with a `testImplementation`, but forcing the concrete version is the cleaner solution IMO.

Reply:
Ok good.